cmd/controller-manager/app/controllers.go

```diff
@@ -281,10 +278,6 @@ func addControllers(
 		controllers["devopscredential-controller"] = devopsCredentialController
 	}
 
-	if storageCapabilityController.IsValidKubernetesVersion() {
```

Comment:
storageCapability supports K8s version below 1.17?

Reply:
en. I consider KS may installed on k8s 1.15 1.16，so i just let snapshot disable on 1.15, 1.16. Other capabilities still work, this may be good for users, and i think it has no problems.